Functions/Assertions/Should.ps1

```diff
@@ -83,11 +84,13 @@ function Should {
             $testFailed = Get-TestResult $parsedArgs $value
 
             if ($testFailed) {
+                $ShouldExceptionLineText = $MyInvocation.Line.TrimEnd("`n")
```

Comment:
The output looks pretty good now.  Do you think we should just call .Trim() instead of only trimming the trailing newlines?  At the moment, the line's indentation is creating some space (which may not be consistent between tests):

```
Describing Testing
 [-] Fails 2.08s
   Expected: {False}
   But was:  {True}
   at line: 3 in C:\Users\dlwya_000\desktop\test.Tests.ps1
   3:         $true | Should Be $false
```